### PR TITLE
Udated and improved README.md

### PR DESCRIPTION
**Pull request by surajphulara**
Corrected grammatical errors and spelling mistakes.

### CODE REVIEW

**Comment by surajphulara** [comment]
> @googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ We use [Hugo](https://gohugo.io/) to format and generate our website, the
 and [Netlify](https://www.netlify.com/) to manage the deployment of the site. 
 Hugo is an open-source static site generator that provides us with templates, 
 content organization in a standard directory structure, and a website generation 
-engine. You write the pages in Markdown, and Hugo wraps them up into a website.
+engine. Write your pages in Markdown and Hugo wraps them up into a website.
```
[the remark this answers is not in the document]
Comment (8bitmp3) [reply README.md:11]: cc @terrytangyuan @Bobgy @jlewi Hacktoberfest

File: README.md [hunk 2]
```diff
@@ -27,27 +27,27 @@ updates:
...
 
-If you need more help with the GitHub workflow, follow this
+If you need more help with the GitHub workflow, follow this link
 [guide to a standard GitHub
 workflow](https://github.com/kubeflow/website/blob/master/quick-github-guide.md). 
 
 ## Previewing your changes on a local website server
 
-If you'd like to preview your doc updates as you work, you can install Hugo
+If you like to preview your doc updates as you work, you may install Hugo
```
Comment (8bitmp3) [inline README.md:45]: No changes here are necessary.

File: README.md [hunk 3]
```diff
@@ -27,27 +27,27 @@ updates:
...
 and run a local server to host your website. This section shows you how.
 
 ### Install Hugo and other dependencies
 
-You need Hugo version 0.60 or later, and it must be the **extended** version of 
+You need Hugo version 0.60 or later and it must be the **extended** version of
```
Comment (8bitmp3) [inline README.md:50]: No changes here are necessary.

**Review by terrytangyuan — COMMENTED** [review]
/close

**Pull request closed without merging**